h/static/scripts/group-forms/components/CreateEditGroupForm.tsx

```diff
@@ -163,17 +183,52 @@ export default function CreateGroupForm() {
     setLocation(response.links.html);
   };
 
+  const updateGroup = async (e: SubmitEvent) => {
+    e.preventDefault();
+    setErrorMessage('');
+    setSaved(false);
+    setSaving(true);
+
+    let response: CreateUpdateGroupAPIResponse;
+
+    try {
+      response = (await callAPI(
+        config.api.updateGroup!.url,
+        config.api.updateGroup!.method,
+        {
+          id: group!.pubid,
+          name,
+          description,
+        },
+      )) as CreateUpdateGroupAPIResponse;
+    } catch (apiError) {
+      setSaving(false);
```

Comment:
The `setSaving(false)` call could be moved into a `finally` block, since such blocks always run, even if the try or catch blocks returns ([docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch#the_finally_block)).

Reply:
Done